### DISCUSSION TITLE
[Question] What do you think about dashboard page.

### DISCUSSION THREAD

**Discussion by piekny27** [piekny27/PulseBook#14] [Q&A] [answered]
[![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
<h3>Q: "What do you think about dashborad page. "</h3>
📋 Describe it and paste your feelings bellow.
🏆 One person with best comment will win follow by Pulsebook team! 🏆

**Comment by adrianbeys** [comment, marked as answer]
I think it is very creatively made. I like the dynamic elements of the workspace.

**Reply by piekny27** [reply]
Congrats! 🏆You win!🏆 Good job!

**Comment by OlexTix** [comment]
I find it very pleasant for the eye. I appreciate the effort put into the graphics :)
